src/tests/functional/navigation_tests.ts

```diff
@@ -136,6 +136,19 @@ test("test following a same-origin POST form button[data-turbo-action=replace]",
   assert.equal(await visitAction(page), "replace")
 })
 
+test("test following a POST form clears cache", async ({ page }) => {
+  await page.evaluate(() => {
+    const cachedElement = document.createElement("some-cached-element")
+    document.body.appendChild(cachedElement)
+  })
+
+  page.click("#form-post-submit")
+  await nextBeat // 301 redirect response
```

Comment:
In the Playwright powered test suite, we need to call the functions that create the `Promise` instance (as opposed to accessing the `Promise` instances through the `this.nextBeat` property).

Similarly, what do you think about prepending an `await` ahead of the `page.click(...)` call on the line above?

Reply:
@seanpdoyle - troubleshooting this right now.  

> In the Playwright powered test suite, we need to call the functions that create the Promise instance (as opposed to accessing the Promise instances through the this.nextBeat property).

Does this simply mean?:
```diff
- await nextBeat
+ await nextBeat()
```
I'm struggling getting the tests to run in my local MacOS environment. 

```
yarn test:browser --project=chrome                                                                                                                                                        

yarn run v1.22.17
$ playwright test --project=chrome

Running 260 tests using 10 workers


Error: Timed out waiting 120000ms from config.webServer.




  260 skipped
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Any tips for correcting this so I can test locally?

Reply:
I haven't experienced that issue before. The local server is still managed by the same code as before. Have you tried `yarn run playwright install` or `yarn run playwright install --with-deps`?

Reply:
yes, I ran `yarn run playwright install --with-deps` first.  Will google some more.

Reply:
For what it's worth, I've cloned your branch locally and the tested these changes' test file:

```sh
❯ yarn test:browser src/tests/functional/navigation_tests.ts
# ...
  Slow test file: [firefox] › navigation_tests.ts (38s)
  Slow test file: [chrome] › navigation_tests.ts (29s)
  Consider splitting slow test files to speed up parallel execution

  76 passed (41s)
✨  Done in 42.06s.
```

Reply:
Tests now report:

```
  Slow test file: [chrome] › form_submission_tests.ts (39s)
  Slow test file: [chrome] › navigation_tests.ts (24s)
  Consider splitting slow test files to speed up parallel execution
```
Did this MR impact cause the test to run slower?

Reply:
No, those are already slowwwwww. That output just happened to be part of what I clipped from my terminal.